packages/lib/modules/pool/actions/add-liquidity/form/AddLiquidityFormTabs.tsx

```diff
@@ -36,7 +37,9 @@ export function AddLiquidityFormTabs({
   const { toCurrency } = useCurrency()
 
   const isDisabledProportionalTab =
-    nestedAddLiquidityEnabled || !supportsProportionalAddLiquidityKind(pool)
+    nestedAddLiquidityEnabled ||
+    !supportsProportionalAddLiquidityKind(pool) ||
+    hasWeightedPool2Tokens(pool)
```

Comment:
Looks like `hasWeightedPool2Tokens` should be within `supportsProportionalAddLiquidityKind`.